terraform/etl/24-aws-glue-housing.tf

```diff
@@ -56,3 +56,38 @@ module "rentsense_output" {
   }
 
 }
+    
+
+module "rentsense_output_test" {
+  source                    = "../modules/aws-glue-job"
+  count                      = !local.is_production_environment ? 1 : 0
+  is_live_environment       = local.is_live_environment
+  is_production_environment = local.is_production_environment
+
+  glue_scripts_bucket_id     = module.glue_scripts.bucket_id
+  glue_temp_bucket_id        = module.glue_temp_storage.bucket_id
+  glue_role_arn              = aws_iam_role.glue_role.arn
+  job_name                   = "${local.short_identifier_prefix}Rentsense outputs"
```

Comment:
Name must be unique, so perhaps add ` test` at the end since this is for testing purposes only for now.